src/auth/Combined.strategy.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { AuthGuard, PassportStrategy } from '@nestjs/passport';
+import { UniqueTokenStrategy } from 'passport-unique-token';
+
+@Injectable()
+export class CombinedStrategy extends PassportStrategy(
+  UniqueTokenStrategy,
+  'combined-strategy',
+) {
+  constructor(private config: ConfigService) {
+    super({});
+  }
+
+  async validate(token: string, done) {
+    if (
+      token != this.config.get('APP_DISCORD_TOKEN') &&
+      token != this.config.get('APP_GITHUB_TOKEN')
+    ) {
```

Comment:
I think this check might be better as an array, rather than app specific

```ts
if (this.config.get('APPROVED_TOKENS').includes(token))
```

Reply:
Okay 👍 
I did this on purpose as explained above.
If we dont have the described scenario thats probably the easier way of doing it.

Reply:
Sorry I got excited and missed your explanation above 🤦‍♂️ . I think keep it simple and only what we need for now would reduce confusion for people joining the project and less for us to test.